src/Components/test/E2ETest/ServerExecutionTests/CircuitGracefulTerminationTests.cs

```diff
@@ -11,6 +11,7 @@
 using Microsoft.AspNetCore.E2ETesting;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging.Testing;
+using Microsoft.AspNetCore.Testing;
```

Comment:
Suggest removing the `using` on line 10 since we tend to keep our `using`s in alphabetical order